docs/community/faq.md

```diff
@@ -4,9 +4,111 @@ title: Community FAQ
+
+# Contact Information
+
+### What is the official Near email?
+Official email hello@nearprotocol.com. However, for proposals and suggestions, please create a post in our forum.
+
+### Where can I sign up to the newsletter?
+You want to receive updates on the latest and greatest in the NEAR ecosystem, then head over to https://nearprotocol.com/newsletter/
+
+# Contributing to NEAR
+
+### How can I get involved?
+It depends on what you are excited about doing. Do you want to develop on Near? Build tutorials? Answer questions? Expand our documentation? We recommend you have a look at our Contributor Program http://nearprotocol.com/contributor. If you are looking for something a little different, or you have questions, please reach out to the team on [Telegram](https://t.me/cryptonear) or [Discord](http://near.chat/).
+
+### Does Near have a bug bounty program available?
+While we currently do not have a bug bounty program, you can contribute to our development of resources through our Contributor Program. Additionally, we have hosted several hackathons; for future events sign-up to our newsletter https://nearprotocol.com/newsletter/
+
+### Are articles (not NEAR-specific) allowed to be shared on Discord or Telegram?
+Links with educational content are allowed to be shared in the channels. If you are uncertain, please message an admin or somebody from the core team first; either on [Telegram](https://t.me/cryptonear) or [Discord](http://near.chat/). 
+
+### Where can I provide input?
+Most of our discussions and change or improvement proposals are shared and discussed on our Forum https://commonwealth.im/near/.
+
+# Token Economics
+
+### What are the basic denominations of NEAR?
+We use “NEAR” as a basic denomination. The plural is “NEAR”.
+1 NEAR has 1E18 atto-NEAR. Or 1000 milliNEAR, 1M microNEAR.
+Total supply of NEAR at launch is 1 billion.
+
+### Is there an EIO, ICO or any other form of token sale?
+Currently, there is no public investment opportunity. NEAR does not have any public plans for a token offering at this stage. We will let you know if and when that changes.
+
+### Are we going to have airdrops?
+We are working with partners across the ecosystem to make sure tokens get in the hands of the right people to make the network successful. When those programs go live, you will be notified through our public channels. 
+
+### When is the MOON?
+Check the lunar calendar yourself. (https://www.timeanddate.com/moon/phases/) 
+
+### Where can I learn more about your token economics?
+Please head over to our economics paper https://nearprotocol.com/papers/economics-in-sharded-blockchain/ 
+
+# Product 
+
+### What products are currently available on NEAR?
+Products include the NEAR-wallet, NEAR-shell, block explorer, IDE, and the actual blockchain (available on TestNet). To learn more, please head over to our documentation.
```

Comment:
Near studio will soon be deprecated. So not sure whether we want to put IDE here

Reply:
Taken out